packages/hosted-form-v2/src/iframe-content/hosted-input-validator.ts

```diff
@@ -38,7 +38,7 @@ export default class HostedInputValidator {
 
             return results;
         } catch (error) {
-            if (error.name !== 'ValidationError') {
+            if (error instanceof Error && error.name !== 'ValidationError') {
```

Comment:
Please use typeguard for this

Reply:
Changed it into the same as the hoste-form's file.